html2asketch/document.js

```diff
@@ -8,18 +8,20 @@ function pageToPageReference(page) {
   };
 }
 
-function textStyleToSharedStyle(textLayer) {
+function textStyleToSharedStyle(textLayer, documentId, i) {
   return {
     '_class': 'sharedStyle',
-    'do_objectID': generateID(),
+    'do_objectID': `${documentId}/textStyles-${i}`,
     name: textLayer._name,
     'style': textLayer._style.toJSON()
   };
 }
 
+const defaultOptions = {id: 'UI KIT'};
```

Comment:
I'd go with `constructor({id = generateID()})` instead. If someone doesn't care about those IDs, lets just generate random ones for them.

Reply:
sure